WordPress/gutenberg
# [Block Library - Query Pagination Next/Previous]: Add an arrow attribute and sync next/previous block's arrow

## Description
Resolves: https://github.com/WordPress/gutenberg/issues/33474

This PR offers users/themes the ability to add an arrow to the Next Page link. The advantage over this approach, rather than just allowing users/themes to set a label, is it means that in the default placeholder state, the "Next Page" text will be translated. It's also easier for users to set an arrow in this way rather than typing an HTML entity that they may not know how to type.

This PR syncs all `Query Pagination Next and Previous` blocks to use the same arrow value.

## Testing instructions
1. Insert Query Pagination block in a Query Loop and change arrows in `Next/Previous` blocks.
2. Observe that they are synced, front end matches editor display and RTL is working properly (mirroring the arrows)


## Screenshots 

https://user-images.githubusercontent.com/16275880/129545682-7805abc3-4939-46cc-b0de-b09b6ecd0ee8.mov



## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
